src/components/layout/AppBar.vue

```diff
@@ -254,6 +293,11 @@ export default class AppBar extends Mixins(StateMixin, ServicesMixin) {
     })
   }
 
+  handleLightToggle () {
+    const target = this.isPinLightON() ? 0 : 1
+    this.sendGcode(`SET_PIN PIN=${this.topNavPinLight} VALUE=${target}`, Waits.onSetOutputPin)
```

Comment:
If `isPinLightON` gets refactored to the `topNavPinLightOn` getter, this can become the following:
```suggestion
    this.sendGcode(`SET_PIN PIN=${this.topNavPinLight} VALUE=${+!this.topNavPinLightOn}`, Waits.onSetOutputPin)
```